server/config.go

```diff
@@ -41,9 +43,11 @@ type LoggingConfig struct {
 }
 
 type Options struct {
-	AppName              string   `yaml:"app_name"`
-	ConfigurationPath    string   `yaml:"configuration_path"`
-	ConfigurationV0Paths []string `yaml:"configuration_v0_paths"`
+	AppName                 string           `yaml:"app_name"`
+	ConfigurationPath       string           `yaml:"configuration_path"`
+	EnableGlobalConfig      bool             `yaml:"enable_global_config"`
+	DefaultRepositoryConfig bulldozer.Config `yaml:"default_repository_config"`
```

Comment:
If you define this as `*bulldozer.Config`, then I think you can remove the `EnableGlobalConfig` flag and make it so that the existence of this options enables the default fallback. That will also simplify the code in `server.go` a bit.